stages/stage_2/optimize.py

```diff
@@ -1,11 +1,12 @@
-from clvm import KEYWORD_TO_ATOM, QUOTE_ATOM
+from clvm import KEYWORD_TO_ATOM
```

Comment:
is your intention to have `clvm` not export `QUOTE_ATOM`? or why is it better to define it by pulling it out of `KEYWORD_TO_AOM`?

Reply:
My main issue is it's redundant with `KEYWORD_TO_ATOM`. Eventually, I'd like to group `assemble`, `disassemble`, `KEYWORD_TO_ATOM` and `KEYWORD_FROM_ATOM` into a class analogous to `Dialect` that hides these details (essentially a `clvm_tools` version of `Dialect`).

Reply:
I think I removed this because I didn't like the asymmetry of how `QUOTE_ATOM` was handled differently from the other operators (although I admit as an operator, it is a bit unique so the asymmetry might be justified). I do like `_ATOM` as a suffix better than `_KW` though, so I should probably change the names to `_ATOM`.